lib/src/importer-registry.ts

```diff
@@ -11,6 +11,28 @@ import {FileImporter, Importer, Options} from './vendor/sass';
 import * as proto from './vendor/embedded_sass_pb';
 import {catchOr, thenOr, PromiseOr} from './utils';
 
+const entryPointDirectoryKey = Symbol();
+
+export class NodePackageImporter {
+  [entryPointDirectoryKey]?: string;
+
+  constructor(entryPointDirectory?: string) {
+    entryPointDirectory = entryPointDirectory
+      ? p.resolve(entryPointDirectory)
+      : require.main?.filename
+      ? p.dirname(require.main?.filename)
+      : undefined;
```

Comment:
Add parens around nested ternary expressions.

Reply:
Prettier strips out parens around nested ternary where they don't impact the end result. Newer versions of Prettier add an [`experimentalTernaries` option](https://prettier.io/docs/en/options#experimental-ternaries), but 1) even that only adjusts the line breaks in this case, and doesn't allow wrapping parens, and 2) our version of Prettier here is locked by [gts](https://github.com/google/gts). Do you prefer wrapping parens here even if it requires an `// eslint-disable` comment?

Reply:
No, if Prettier insists on this I suppose it's fine.